app/templates/components/edit-reminder.hbs

```diff
@@ -4,3 +4,4 @@
 {{#link-to "reminders.reminder" model.id}}
   <button class="spec-save-button" {{action "save"}}>save</button>
 {{/link-to}}
+<button class="spec-reset-button" {{action "reset"}}>reset</button>
```

Comment:
@martensonbj the attributes are never dirty i believe.
<img width="1280" alt="screen shot 2017-02-15 at 11 28 42 am" src="https://cloud.githubusercontent.com/assets/20427352/22989148/fac0c496-f371-11e6-9c0c-5829ea89e590.png">